src/intervals/functions.jl

```diff
@@ -84,7 +84,7 @@ end
         hi = hull(hi, hi1)
     else
         lo, hi = with_bigfloat_precision(53) do
-            aa = @biginterval(a)
+            aa = Interval(big(a.lo), big(a.hi)) # before: @biginterval(a)
```

Comment:
If `a` is a `BigFloat` interval, we don't want to restrict its precision to 53 bits.